Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ARG BASE=golang:1.16-alpine3.12
+ARG BASE=golang:1.16-alpine3.14
```

Comment:
We need to change `FROM scratch` to `FROM alpine:3.14` below. This is required for security-bootstrapper